packages/cli/src/modules/new/lib/defaultTemplates.ts

```diff
@@ -16,6 +16,7 @@
 
 export const defaultTemplates = [
   '@backstage/cli/templates/frontend-plugin',
+  '@backstage/cli/templates/frontend-plugin-module',
```

Comment:
Thinking we should leave this out of the default set for now, but I can update that in a followup